src/irmin-pack/IO.ml

```diff
@@ -106,7 +106,7 @@ module Unix : S = struct
     mutable flushed : int64;
     readonly : bool;
     version : version;
-    buf : Buffer.t;
+    mutable buf : string list;
```

Comment:
Looks like an unrelated change here. I got the impression that this ended up being bad for performance?

Reply:
hum yes indeed I though I had reverted this but that doesn't seem to be the case. Thanks for noticing!